src/backend/distributed/commands/vacuum.c

```diff
@@ -108,21 +100,26 @@ PostprocessVacuumStmt(VacuumStmt *vacuumStmt, const char *vacuumCommand)
 		StartTransactionCommand();
 	}
 
-	/*
-	 * Here we get the relation list again because we might have
-	 * closed the current transaction and the memory context got reset.
-	 * Vacuum's context is PortalContext, which lasts for the whole session
-	 * so committing/starting a new transaction doesn't affect it.
-	 */
-	citusRelationIdList = VacuumCitusRelationIdList(vacuumStmt, vacuumParams);
-	bool distributedVacuumStmt = IsDistributedVacuumStmt(vacuumParams.options,
-														 citusRelationIdList);
-	if (!distributedVacuumStmt)
-	{
-		return;
+	if(vacuum_all){
```

Comment:
If this `vacuum_all` is just for here, can you just replace it with `list_length(vacuumStmt->rels) == 0`?
And if not, can you use camel case, `vacuumAll`?